Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -1,199 +1,207 @@
+                "About": "Offset of XAML elements using composition",
+                "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Animations/Behaviors",
+                "CodeFile": "OffsetBehaviorCode.bind",
+                "XamlCodeFile": "OffsetBehaviorXaml.bind",
+                "Icon": "/SamplePages/Offset/offsetBehavior.png"
+            },
+            {
+                "Name": "Rotate",
+                "Type": "RotateBehaviorPage",
+                "About": "Rotation on XAML elements using composition",
+                "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Animations/Behaviors",
+                "CodeFile": "RotateBehaviorCode.bind",
+                "XamlCodeFile": "RotateBehaviorXaml.bind",
+                "Icon": "/SamplePages/Rotate/rotateBehavior.png"
+            },
+            {
+                "Name": "Blur",
+                "Type": "BlurBehaviorPage",
+                "About": "Blur XAML elements using composition",
+                "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Animations/Behaviors",
+                "CodeFile": "BlurBehaviorCode.bind",
+                "XamlCodeFile": "BlurBehaviorXaml.bind",
+                "Icon": "/SamplePages/Blur/blurBehavior.png"
+            }
+        ]
+    },
+    {
+        "Name": "Services",
+        "Icon": "Icons/Services.png",
+        "Samples": [
+            {
+                "Name": "Bing Service",
+                "Type": "BingPage",
+                "About": "The Bing Service allows you to retrieve Microsoft Bing web search engine results.",
+                "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Animations/Behaviors",
+                "CodeFile": "BingCode.bind",
+                "Icon": "/SamplePages/Bing Service/icon.png"
+            },
+            {
+                "Name": "Facebook Service",
+                "Type": "FacebookPage",
+                "About": "The Facebook Service allows you to retrieve or publish data to Facebook graph.",
+                "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.Services/Services/Facebook",
+                "CodeFile": "FacebookCode.bind",
+                "Icon": "/SamplePages/Facebook Service/FacebookLogo.png"
+            },
+            {
+                "Name": "Twitter Service",
+                "Type": "TwitterPage",
+                "About": "The Twitter Service allows you to retrieve or publish data to Twitter.",
+                "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.Services/Services/Twitter",
+                "CodeFile": "TwitterCode.bind",
+                "Icon": "/SamplePages/Twitter Service/TwitterLogo.png"
+            }
+        ]
+    }
 ]
```

Comment:
I'm assuming something went wrong with your line endings. There shouldn't be this many changes